### ISSUE TITLE
A problem about elements

### ISSUE THREAD

**Issue by ShiguangSun** [fdeng18/dreamer-pro#5]
Hi, I configured the conda environment as requirements.txt, but I encountered the following error when running the experiment:
Traceback (most recent call last):
  File "dreamerv2/train.py", line 35, in <module>
    parsed, remaining = elements.FlagParser(configs=['defaults']).parse_known(
AttributeError: module 'elements' has no attribute 'FlagParser'
Any suggestions?

**Comment by fdeng18** [comment]
Thanks for your interest in our work! There are quite a few recent updates to the `elements` package. I would suggest trying `elements==0.3.2`. Please let me know if it works.

**Comment by ShiguangSun** [comment]
Yes, it works! Thanks. But now there is another problem. If I use GPU to train, an assertion error will occur:
assert tf.config.experimental.list_physical_devices('GPU'), message
AssertionError: No GPU found. To actually train on CPU remove this assert.

It seems that I can train normally with CPU. why is that? Is it related to the versions of cudatoolkit and cudnn?

**Comment by fdeng18** [comment]
I used cudatoolkit 11.2 and cudnn 8.2.1. Maybe you could try these versions.

**Comment by ShiguangSun** [comment]
I am encountering an issue during installation where it indicates a version conflict between CUDA Toolkit 11.2 and CuDNN 8.2.1.  That's strange. I have another question. How to choose the number of pixels for translation in data augmentation? It seems to be 3 in the code. How is this selected?

**Comment by fdeng18** [comment]
The random shift follows [DrQ-v2](https://arxiv.org/abs/2107.09645). It shifts 84x84 images by 4 pixels. We use 64x64 images, so we reduce the random shift pixels proportionally.

For installation, maybe you could try cudatoolkit 11.2 and cudnn 8.1.0.

**Comment by ShiguangSun** [comment]
Thanks, I will try cudatoolkit 11.2 and cudnn 8.1.0.  And I want to know about the ame_across_time in andom_translate function. The default setting is True. If it is False, will it affect performance?

**Comment by fdeng18** [comment]
Setting same_across_time to False will change the environment dynamics. It will add more randomness to the dynamics and make it harder for the world model to predict the future. The performance is expected to be worse, though we did not do an ablation study on this.

**Comment by ShiguangSun** [comment]
Thanks very much！

**Comment by ShiguangSun** [comment]
Hi, I have a new question here. When I checked the results, I found that tensorboard visualized the video during the eval, but in the code, I did not see any training for a decoder. How was this recorded video generated?

**Comment by fdeng18** [comment]
I think it's just the observation from the environment. See https://github.com/fdeng18/dreamer-pro/blob/main/DreamerPro/dreamerv2/train.py#L111.

**Comment by ShiguangSun** [comment]
Thanks！
